app/connector/UserDetailsConnector.scala

```diff
@@ -32,20 +30,22 @@ trait UserDetailsConnector {
   def getUserDetails(userDetailsUri: String)(implicit hc: HeaderCarrier) = http.GET[UserDetails](userDetailsUri)
 }
 
-object CredentialRole extends Enumeration {
-  type CredentialRole = Value
-  val User, Unknown = Value
+case class CredentialRole(value: String) extends AnyVal {
```

Comment:
@paoloboni: 
I'm not sure about adding the `val`. I can see in the [Scala documentation](http://docs.scala-lang.org/overviews/core/value-classes.html) that they use `val`, 
however I see [here](http://stackoverflow.com/questions/17658301/scala-can-case-class-with-one-field-be-a-value-class) that `val` is not needed for `case class` as it will compile to the same bytecode.